packages/payment-integration-api/src/contexts/checkout-context/StyleContext.tsx

```diff
@@ -2,6 +2,7 @@ import { createContext, useContext } from 'react';
 
 export interface StyleContextProps {
     newFontStyle: boolean;
+    themeV2: boolean;
```

Comment:
I am not sure why this is in payment integration package. I believe this should be in ui or some other folder 🤔 Also shall we rename this to `ThemeContext`?

Reply:
Renaming this and related files will touch over 60 files.
Perhpas it's easier for review if we do it in another PR.

Reply:
Sounds good 👍